### PR TITLE
[9.0] [MIG] sale_order_price_recalculation module

### PR DESCRIPTION
**Pull request by cubells**
- Updated README
- Updated views
- Updated methods

cc @Tecnativa

### CODE REVIEW

File: sale_order_price_recalculation/__openerp__.py [hunk 1]
```diff
@@ -1,24 +1,27 @@
...
-    "version": "8.0.1.0.0",
-    "depends": [
-        "sale",
-    ],
-    'license': 'AGPL-3',
+    "version": "9.0.1.0.0",
+    "category": "Sales Management",
+    "website": "http://github.com/OCA/sale-workflow",
     "author": "AvanzOSC,"
               "Serv. Tecnol. Avanzados - Pedro M. Baeza,"
```
Comment (pedrobaeza) [inline sale_order_price_recalculation/__openerp__.py]: Remove my old company

**Review by pedrobaeza — APPROVED** [review]
Little comment, but :+1:

**Comment by pedrobaeza** [comment]
Runbot warning unrelated

**Comment by luismontalba** [comment]
I go to Sales > Sales Order
I create a new one:

seleccion_001

I choose a customer and go to add products:

seleccion_002

I choose a product and try to save:

seleccion_003

And I get this error:

seleccion_004

**Comment by chienandalu** [comment]
no runbot?

**Comment by chienandalu** [comment]
Same error as @luismontalba for me

**Comment by pedrobaeza** [comment]
@chienandalu @luismontalba you can try now again because the branch was outdated.

File: sale_order_price_recalculation/README.rst [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
 ===================================================
 Recalculation of sales order lines prices on demand
 ===================================================
 
-.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-    :alt: License: AGPL-3
-
 This module add 2 buttons on sale orders (below sale order lines) that:
 
  * recalculates the prices of the order lines that contain a product in them.
```
Comment (luismontalba) [inline sale_order_price_recalculation/README.rst]: * reset descriptions of the products in order lines.

**Review by luismontalba — CHANGES_REQUESTED** [review]

**Comment by luismontalba** [comment]
I have tested it on runbot and both buttons works fine, but when I click on the "Modify Variants" icon in the order line:

seleccion_001

I get the Javascript related error:

seleccion_002

**Comment by pedrobaeza** [comment]
But this in unrelated to this module. It's documented in its corresponding module: https://github.com/OCA/sale-workflow/blob/9.0/sale_order_variant_mgmt/README.rst#known-issues--roadmap

**Review by luismontalba — APPROVED** [review]
So I think the functionality of the module is correct !

**Review by chienandalu — APPROVED** [review]

**Comment by chienandalu** [comment]
tested on runbot

**Pull request merged**